benchmarks/gpu_benchmarks.ipynb

```diff
@@ -0,0 +1,1494 @@
+{
```

Comment:
<p>Did the results chage? I thought JAX on GPU was faster</p><p> </p><p>Also, the y ticks are hard to read ;)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/py-econometrics/pyfixest/pull/784/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>On the y-ticks - I was hoping they'd get easier to read one the benchmarks ran for longer (though am not sure if seaborn actually supports this type of auto-scaling, but I would suppose so?)</p>